inst/WORDLIST

```diff
@@ -42,3 +42,5 @@ usethis
 webpack
 yml
 Żyła
+nvm
+PowerShell
```

Comment:
Did you use `spelling::update_wordlist()` to update it? I'm surprised that these landed out of alphabetical order.

Reply:
Ah, no I didn't. Honestly assumed that it was meant to be edited by hand, I will fix this

Reply:
Fixed